### DISCUSSION TITLE
如何设置才能让其他人挂载alist v3时需要输入 Access token

### DISCUSSION THREAD

**Discussion by imopro** [AlistGo/alist#2814] [General]
在添加 Driver 中的 alist V3 配置中有 Access token项目，但不是必填的。那么这个Access token是要如何启动呢？也就是当对方挂载我方时，需要填写 Access token 才可以。这个要如何配置？
另外，可以拒绝其他人挂载alist吗？这是需要修改源码吗？有无相关文档说明呢？
谢谢各位大神啦。

**Comment by NovaShen0724** [comment]
试试加密码？加密码是不是就得用token了
